arm/Microsoft.Network/firewallPolicies/deploy.bicep

```diff
@@ -0,0 +1,189 @@
+@description('Required. Name of the Firewall Policy.')
```

Comment:
Please add a status badge of the pipeline running to the PR description

Reply:
[![Network: FirewallPolicies](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.firewallpolicies.yml/badge.svg)](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.firewallpolicies.yml)

Reply:
I meant the PR description, not this comment :D Will add it there for you